ts/tsconfig.json

```diff
@@ -2,8 +2,8 @@
   "include": ["./src/**/*"],
   "compilerOptions": {
     "moduleResolution": "node",
-    "module": "es6",
-    "target": "es2019",
+    "module": "es2020",
+    "target": "es2020",
```

Comment:
`es2020` is required to support dynamic imports. 

I think Anchor may also want to drop support for CJS. [Solana Wallet Adapter](https://github.com/solana-labs/wallet-adapter) doesn't support CJS so it would be consistent w/ the rest of the Solana ecosystem. 

[Other developers](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c) are moving to only ESM because it's really confusing and hard to support both CJS and ESM properly. 

(We actually use CJS at Luma and are in the process of migrating over to ESM.)